common4j/src/main/com/microsoft/identity/common/java/commands/DeviceCodeFlowCommand.java

```diff
@@ -36,6 +33,8 @@
 import java.util.Date;
 import java.util.concurrent.TimeUnit;
 
+import lombok.NonNull;
```

Comment:
are the  import statements here required?

Reply:
yes, since we are using `@NonNull` annotation in the cosntructor